nipype/interfaces/utility/base.py

```diff
@@ -303,15 +301,11 @@ def _rename(self):
 
     def _run_interface(self, runtime):
         runtime.returncode = 0
-        _ = copyfile(self.inputs.in_file,
-                     os.path.join(os.getcwd(), self._rename()))
+        out_file = os.path.join(runtime.cwd, self._rename())
+        _ = copyfile(self.inputs.in_file, out_file)
+        self._results['out_file'] = out_file
```

Comment:
Now that the constructor of `SimpleInterface` is not called, then `self._results` should be initialized in `__init__` as an empty dict, shouldn't it?

Reply:
Scratch that, I see that you are actually adding the `SimpleInterface`. For some reason I read the diff in the opposite way.